blog_system/settings.py

```diff
@@ -0,0 +1,132 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-u=t)fl8+(ukd%xpu^ntx*fa0^9dle^j((^*w%o+o8(q(g@6c!u"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
The `DEBUG` setting is `True`, which is not recommended for production environments. Ensure this is set to `False` in production to avoid exposing sensitive information.

Reply:
we do it for learn